package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint --ext js,ts --quiet \"src/**/*.{js,ts}\"",
     "lint-report": "eslint --ext js,ts -f json -o eslint-report.json \"src/**/*.{js,ts}\"",
     "test": "npm test --prefix src/common/latest && npm test --prefix src/common/sonarqube-v4",
-    "test-ci": "npm run test-ci --prefix src/common/latest && npm run test-ci --prefix src/common/sonarqube-v4",
+    "test-ci": "npm run test --prefix src/common/latest && npm run test --prefix src/common/sonarqube-v4",
```

Comment:
the old ci task differentiated only by `--coverage`, is this the problem with missing coverage?

Reply:
For some reason, when I switched to `test-ci`, it fails the `Azure Pipelines` CI, that's why I've dropped it, I'm investigating why atm 😢